### PR TITLE
Fix: Unlock mouse on barn tp

### PR DESCRIPTION
**Pull request by not-a-cowfr**
## What
Will now unlock mouse when tping to barn (`/plottp barn`) instead of just for plots

## Changelog Fixes
+ Fixed the mouse not unlocking when teleporting to the Barn. - not_a_cow

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/features/misc/LockMouseLook.kt [hunk 1]
```diff
@@ -32,7 +39,8 @@ object LockMouseLook {
 
     @SubscribeEvent
     fun onChat(event: LorenzChatEvent) {
-        if (!event.message.startsWith("§aTeleported you to §r§aPlot")) return
+        println(!unlockMousePattern.matches(event.message))
```
Comment (NopoTheGamer) [inline src/main/java/at/hannibal2/skyhanni/features/misc/LockMouseLook.kt]: 🔥

Comment (not-a-cowfr) [reply src/main/java/at/hannibal2/skyhanni/features/misc/LockMouseLook.kt]: i have no memory of even putting that there

File: src/main/java/at/hannibal2/skyhanni/features/misc/LockMouseLook.kt [hunk 2]
```diff
@@ -9,12 +9,19 @@ import at.hannibal2.skyhanni.features.garden.SensitivityReducer
...
+import at.hannibal2.skyhanni.utils.RegexUtils.matches
 import net.minecraft.client.Minecraft
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
 
 @SkyHanniModule
 object LockMouseLook {
 
+    private val unlockMousePattern by RepoPattern.pattern(
+        "unlockmouse",
+        "§aTeleported you to (.*)",
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/features/misc/LockMouseLook.kt]: Why is there a capture group, either remove it or name it

File: src/main/java/at/hannibal2/skyhanni/features/misc/LockMouseLook.kt [hunk 3]
```diff
@@ -9,12 +9,19 @@ import at.hannibal2.skyhanni.features.garden.SensitivityReducer
...
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import at.hannibal2.skyhanni.utils.RegexUtils.matches
 import net.minecraft.client.Minecraft
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
 
 @SkyHanniModule
 object LockMouseLook {
 
+    private val unlockMousePattern by RepoPattern.pattern(
+        "unlockmouse",
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/features/misc/LockMouseLook.kt]: This repo pattern key is not descriptive enough

**Review by CalMWolfs — CHANGES_REQUESTED** [review]
In addition to these two issues, this also changes the functionality of the code. Now when i warp to plot 1 it also unlocks my mouse

**Comment by not-a-cowfr** [comment]
> In addition to these two issues, this also changes the functionality of the code. Now when i warp to plot 1 it also unlocks my mouse

im not quite sure what you mean, it should be unlocking your mouse when you warp to plot 1

**Comment by CalMWolfs** [comment]
> > In addition to these two issues, this also changes the functionality of the code. Now when i warp to plot 1 it also unlocks my mouse
> 
> im not quite sure what you mean, it should be unlocking your mouse when you warp to plot 1

nvm i read it wrong

**Review by CalMWolfs — APPROVED** [review]

**Pull request merged**